GIT

claudiorave opened the pull request:
7. Create a Pull Request against upstream master branch to include your group name and members in the README.md file. The PR must not have any conflicts. Every participant must create its own PR including his/her complete name and nickname.

claudiorave committed:
GIT
7. Create a Pull Request against upstream master branch to include your group name and members in the README.md file. The PR must not have any conflicts. Every participant must create its own PR including his/her complete name and nickname.

ilanrosenfeld7 commented:
El nombre del PR debería ser más descriptivo

ilanrosenfeld7 commented on README.md:
```diff
@@ -1,6 +1,8 @@
+﻿# Globant Java Bootcamp 2018
 ## Spring - REST - Git - Maven - Testing - RDB
 
+Group-1: Raverta, Claudio Marco (claudiorave). Perez, Jefferson.
```
"Every participant must create its own PR including his/her complete name and nickname."
Que figure solo tu nombre. Y Jefferson debería hacer su propio PR

ilanrosenfeld7 replied:
Chequear que el PR de giorgio ya fue mergeado. Incluir esto bajo su comentario

claudiorave committed:
Fixed issues from the last PR:

1- Groupmate's name no longer included.
2- Pulled from upstream. Additions by group-3 are contemplated.

claudiorave commented:
Ahí lo arregle!
